modules/mod_acl_user_groups/support/acl_user_groups_checks.erl

```diff
@@ -623,6 +623,12 @@ is_owner(insert_rsc, _Context) ->
 is_owner(Id, Context) ->
     is_owner(Id, m_rsc:p_no_acl(Id, creator_id, Context), Context).
 
+is_owner(Id, undefined, #context{user_id=undefined, session_id=SessionId} = Context) ->
+    case m_rsc:p_no_acl(Id, session_owner, Context) of
+        undefined -> false;
```

Comment:
Ouch, I guess you are right about publishing the session-id to the whole world this way. I'll look into your suggestion of storing the RscId's in the session process.